openpype/hosts/houdini/api/lib.py

```diff
@@ -397,8 +401,20 @@ def read(node):
 
     """
     # `spareParms` returns a tuple of hou.Parm objects
-    return {parameter.name(): parameter.eval() for
-            parameter in node.spareParms()}
+    data = {}
+    for parameter in node.spareParms():
+        value = parameter.eval()
+        # test if value is json encoded dict
+        if isinstance(value, six.string_types) and \
+                len(value) > 0 and value.startswith("JSON:::"):
```

Comment:
```suggestion
        if isinstance(value, six.string_types) and value.startswith("JSON:::"):
```
No need to check length, right?

Also shouldn't the `"JSON:::"` prefix be hardcoded elsewhere? Feels a bit arbitrary having this hardcoded here?